cmake/modules/dependency_notifier.cmake

```diff
@@ -50,7 +50,7 @@ if(NOT libuuid_FOUND)
     DEB: apt-get install uuid-dev
     RPM: yum install libuuid
    If you have already installed this package in a nonstandard location 
-   please specify the location by defining the variable LIBJSON-C_ROOT in 
+   please specify the location by defining the variable UUID_ROOT in 
```

Comment:
We have the finder looking for at `LIBUUID_ROOT`
https://github.com/OPAE/opae-sdk/blob/de5d378bd534311b393470771d7ad499a29dd1b4/cmake/modules/FindUUID.cmake#L17